charts/kong/templates/_helpers.tpl

```diff
@@ -21,6 +21,12 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{- define "kong.certificate.issuer" -}}
+{{- $default := printf "%s-%s" .Release.Name "kong-selfsigned-issuer" -}}
+{{- $name := default $default .Values.certificate.proxy.issuer -}}
+{{- printf "%s" $name -}}
+{{- end -}}
```

Comment:
This should be possible to condense to a single line using parentheses to contain the components, which will make it a bit easier to read what the final result is. I think, roughly, `{{- default (printf "%s-%s" .Release.Name "kong-selfsigned-issuer") .Values.certificate.proxy.issuer -}}`, though not positive.

That said, see elsewhere in the Issuer template re using it consistently.

Reply:
done